2019-May-Season/SoftUni-Information-Services/src/SIS.HTTP/Enums/HttpRequestMethod.cs

```diff
@@ -2,9 +2,9 @@
 {
     public enum HttpRequestMethod
     {
-        Get,
-        Post,
-        Put,
-        Delete
+        Get = 1,
+        Post = 2,
+        Put = 3,
+        Delete = 4
```

Comment:
No need (for now probably) to explicitly number the members of the enumeration. Also you can just set the first one to "1" for example the ones below it will increment themselves by one automatically.